actions/hls_sponge/hw/hls_checksum.cpp

```diff
@@ -493,10 +493,10 @@ int main(void)
 			    &Action_Register, &Action_Config);
 	printf(" ==> 2 test calls : checksum = %016llx",
 			(long long) Action_Register.Data.chk_out);
-	if (Action_Register.Data.chk_out == 0x2ccef6d61b67ad2f)
+	if (Action_Register.Data.chk_out == 0x83f5b14f2a6a5d8e)
```

Comment:
I think putting this in tables is nicer to extend at the end. And that is already obvious if you look at 499, where you copy-paste it already in this patch.